### PR TITLE
Add getSetPoint to request -> core

### PR DESCRIPTION
**Pull request by greenblitz4590**

### CODE REVIEW

**Review by YHerm — DISMISSED** [review]

**Review by YHerm — APPROVED** [review]

File: gradlew
Comment (hillelv) [inline gradlew:1]: image

**Review by hillelv — APPROVED** [review]

**Review by Nitay4 — APPROVED** [review]
gg

File: gradlew
[the remark this answers is not in the document]
Comment (drorta) [reply gradlew:1]: I did this on Nitay's laptop, and we couldn't run spotless on it so we did chmod +x gradlew, that makes gradlew executable so we can run spotless, I've seen it happen before and it doesn't change everything from what I know

[the remark this answers is not in the document]
Comment (emilyyyylime) [reply gradlew:1]: yeah should be fine afaik

[the remark this answers is not in the document]
Comment (emilyyyylime) [reply gradlew:1]: It is weird that github shows it as an "empty file" though. If you click "view file" it sends you to the proper contents of it (obviously not written by us so doesn't require review)

[the remark this answers is not in the document]
Comment (drorta) [reply gradlew:1]: It isn't shown as an empty file, if u look it threw a computer (I'm on my phone now so I can't screenshot) you see something like 10744 -> 10745 (the addition of the executable)

[the remark this answers is not in the document]
Comment (emilyyyylime) [reply gradlew:1]: I am referring to this, as shown on a desktop in the Files changed view:
size 0
"Empty file." text

[the remark this answers is not in the document]
Comment (drorta) [reply gradlew:1]: Yeah I understand I refferd to this

[the remark this answers is not in the document]
Comment (emilyyyylime) [reply gradlew:1]: that is not the file size, it's the mode (permission) bits.

[the remark this answers is not in the document]
Comment (drorta) [reply gradlew:1]: Should be fine?

**Review by bossyshrimp68 — APPROVED** [review]

File: gradlew
[the remark this answers is not in the document]
Comment (emilyyyylime) [reply gradlew:1]: Never said otherwise. Just said it's weird that's what GH shows.

[the remark this answers is not in the document]
Comment (drorta) [reply gradlew:1]: ok thanks

**Pull request merged**